Cargo.toml

```diff
@@ -49,7 +49,7 @@ serde = { version = "1.0.101", features = ["derive", "rc"] }
 serde_json = "1.0.45"
 serde_yaml = "0.8.11"
 structopt = "0.3.2"
-symbolic = { git = "https://github.com/getsentry/symbolic", branch = "fix/demangle-fixes", features = ["common-serde", "demangle", "minidump-serde", "symcache"] }
+symbolic = { git = "https://github.com/getsentry/symbolic", branch = "master", features = ["common-serde", "demangle", "minidump-serde", "symcache"] }
```

Comment:
TODO: figure out where this should be pointed.

Reply:
:-D merge master into that branch… and yes, I should push harder on actually landing those fixes in released versions, or some of those we can actually put into symbolic itself, without requiring a patched dependency